firmware/components/driver_sensor_am2320/Kconfig

```diff
@@ -0,0 +1,19 @@
+menu "Driver: AM2320 I2C temp/hum sensor"
+	config DRIVER_AM2320_ENABLE
+		depends on DRIVER_I2C_ENABLE
+		bool "Enable the AM2320 I2C driver (Will set I2C clock to 100khz!)"
+		default n		
+
+	config DRIVER_AM2320_I2C_ADDRESS
+		depends on DRIVER_AM2320_ENABLE
+		hex "I2c address for the AM2320 sensor"
+		default 0x5C
+		select I2C_MASTER_FREQ_HZ
+
+	# This is used to force the already set clock value to 100khz
+	config I2C_MASTER_FREQ_HZ
```

Comment:
Please do not force the I2C interface speed, let the user enter what they want: we can always error out on the existing CONFIG_I2C_MASTER_FREQ_HZ define of the I2C interface itself during compilation.

Reply:
Removed and added error macro to fail compile if wrong value is set